default.nix

```diff
@@ -1,9 +1,10 @@
-{ nixpkgs ? import ./nixpkgs.nix
-, pkgs ? import nixpkgs {}
-}:
+{ nixpkgs ? import ./nixpkgs.nix, pkgs ? import nixpkgs { }, version ? "dev" }:
```

Comment:
So this one won't break. Guess that's okay. It's name in deployments today is just "shelob" (no version)